x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/webhook.tsx

```diff
@@ -473,8 +473,6 @@ const WebhookParamsFields: React.FunctionComponent<ActionParamsProps<WebhookActi
         error={errors.body}
       >
         <EuiCodeEditor
-          fullWidth
```

Comment:
Does it mean that EuiCodeEditor cannot be highlighted if it has invalid value?

Reply:
Correct; EUI doesn't treat the code editor as part of the more traditional inputs. You can open an issue over in [EUI](https://github.com/elastic/eui/issues) to start a discussion for an error/invalid state, I can definitely see the rational for wanting it.